HaskellNet-SSL.cabal

```diff
@@ -40,10 +40,11 @@ library
                        bytestring,
                        data-default
   else
-    build-depends:     base >= 4 && < 5,
-                       HaskellNet >= 0.3 && < 0.6,
-                       tls >= 1.2 && < 1.5,
-                       connection >= 0.2.7 && < 0.3,
-                       network >= 2.4 && < 2.9,
+    build-depends:     base >= 4,
+                       HaskellNet >= 0.3,
+                       tls >= 1.2,
+                       connection >= 0.2.7,
+                       network >= 2.4,
+                       network-bsd >= 2.8,
```

Comment:
Why did you remove the upper bounds?

Reply:
No particular reason. Since we're using `stack` and `LTS`, I never use any bounds, to be honest.
Do you think they should be restored?

Reply:
I think the version constriants shouldn't be removed. This is a Cabal-based project, and not a Stack-based project.

Reply:
Restored